ui/resolution_selection.js

```diff
@@ -555,6 +554,28 @@ shaka.ui.ResolutionSelection = class extends shaka.ui.SettingsMenu {
   }
 
 
+  /**
+   * @param {number} bandwidth
+   * @return {string}
+   * @private
+   */
+  getTextFromBandwidth_(bandwidth) {
+    if (bandwidth > 1e6) {
+      const whole = Math.floor(bandwidth / 1e6);
```

Comment:
Maybe instead of breaking into whole number and single decimal digit, we should consider toFixed().  Try this in dev tools:

```
bandwidth = 2123456;
(bandwidth / 1e6).toFixed(1);  // yields the string '2.1'
```

Unless it's important to you to use '2' instead of '2.0'.